paper-tooltip.html

```diff
@@ -157,24 +158,26 @@
       },
 
       attached: function() {
-        this._target = this.target;
+        var target = this.target;
 
-        this.listen(this._target, 'mouseenter', 'show');
-        this.listen(this._target, 'focus', 'show');
-        this.listen(this._target, 'mouseleave', 'hide');
-        this.listen(this._target, 'blur', 'hide');
+        this.listen(target, 'mouseenter', 'show');
+        this.listen(target, 'focus', 'show');
+        this.listen(target, 'mouseleave', 'hide');
+        this.listen(target, 'blur', 'hide');
       },
 
       detached: function() {
-        if (this._target) {
-          this.unlisten(this._target, 'mouseenter', 'show');
-          this.unlisten(this._target, 'focus', 'show');
-          this.unlisten(this._target, 'mouseleave', 'hide');
-          this.unlisten(this._target, 'blur', 'hide');
-        }
+        var target = this.target;
```

Comment:
Why is this needed? When the tooltip gets re-attached, the `target` is re-computed again anyway.